Thread: add non-Home-Assistant-brand adapter to HA (#42041)
home-assistant/home-assistant.github.io: source/_integrations/thread.markdown

   ![image](/images/integrations/thread/thread-google-br.png)

5. To enable Thread support, you need to install the **OpenThread Border Router** add-on. Follow the corresponding procedure:
   - [Enable Thread on Home Assistant Yellow](https://support.nabucasa.com/hc/en-us/articles/25742476767517).
   - [Enable Thread on Home Assistant Connect ZBT-1](https://support.nabucasa.com/hc/en-us/sections/26122472719517).
   - [Enable Thread on Home Assistant Connect ZBT-2](https://support.nabucasa.com/hc/en-us/sections/31260019451421).
   - [Adding a Thread adapter to Home Assistant](#adding-a-thread-adapter-to-home-assistant).
   - **Result**: The network now shows as the preferred network, joined with the third-party network.

   ![image](/images/integrations/thread/thread-ha-preferred.png)
   - 🎉 You successfully created a Home Assistant Thread network and joined it with a pre-existing third-party network.

## Adding a Thread adapter to Home Assistant

> Follow these steps if you want to use a non-Home-Assistant-brand Thread adapter with Home Assistant.

Reviewer: I think adding a dash between "Home Assistant" isn't really correct. Some possible alternatives:

Perhaps just say "third-party Thread adapter"?
```suggestion
Follow these steps to use a third-party Thread adapter with Home Assistant.
```

Or if we want to stay with the "Home Assistant" Thread adapter:
```suggestion
Follow these steps to set up a non-Home Assistant-branded Thread adapter with Home Assistant.
```

```suggestion
Follow these steps to set up a non-Home Assistant Thread adapter with Home Assistant.
```

The "with Home Assistant" at the end might even be unnecessary. It should be clear we want to do that when looking at HA docs.

> If you have a Home Assistant brand Thread adapter, follow the corresponding instructions instead:

Reviewer: Here, I'd actually add a dash after "Home Assistant" (and also use "branded", I think that reads better):
```suggestion
If you have a Home Assistant-branded Thread adapter, follow the corresponding instructions instead:
```

- [Enable Thread on Home Assistant Yellow](https://support.nabucasa.com/hc/en-us/articles/25742476767517).
- [Enable Thread on Home Assistant Connect ZBT-1](https://support.nabucasa.com/hc/en-us/sections/26122472719517).
- [Enable Thread on Home Assistant Connect ZBT-2](https://support.nabucasa.com/hc/en-us/sections/31260019451421).

### Prerequisites

- [Home Assistant Operating System](/docs/glossary/#home-assistant-operating-system) installed on your smart home hub. For example:
  - On a Home Assistant Green, where Home Assistant OS is preinstalled
  - On a Home Assistant Yellow or on a Raspberry Pi
- Latest updates installed
- A new Thread adapter and a USB extension cable
  - If your adapter supports multiple protocols:
    - They might have another firmware installed by default (Zigbee, for example).
    - Check their documentation and install the OpenThread firmware on the adapter.
    - Check their documentation and take a note of the baudrate.

### To add a Thread adapter to the OpenThread Border Router add-on

1. Plug the adapter into the extension cable and plug it into the Home Assistant hub.

> 2. Go to {% my supervisor_addon title="**Settings** > **Add-ons** > **OpenThread Border Router**" addon="core_openthread_border_router" %} and select the **Configuration** tab.

Reviewer: The OTBR addon isn't installed by default, so we'd need to add a step on how to install that, assuming the user didn't install it before.

3. Under **Devices**, select your adapter.
4. Enter the **Baudrate** as specified in the documentation of your adapter.
   - If you can't find the baudrate, try `460800` or contact the manufacturer's support.
   - **Save** your changes.
   - **Troubleshooting**:
     - Check the logs.
     - If the add-on crashes or fails to communicate with the Thread integration: Toggle the **Hardware flow control** option and try again.
5. Restart the add-on and check the logs. Wait.

>    - **Info**: Your Thread network is managed by Home Assistant. It is not stored on the adapter. Migrating the adapter means that Home Assistant will start to use your new adapter's radio instead of the old one.

Reviewer: We mention "migrating" and "new adapter" here. I don't think we should really do that when the user is just setting up a Thread adapter for the first time. I'd probably just remove that info section entirely from here.
```suggestion
```

6. Go to {% my integrations title="**Settings** > **Devices & services**" %}, select the **Thread** integration.
   - Select the cogwheel {% icon "mdi:cog-outline" %}.

>    - **Result**: You should now see that the Thread network is still there.

Reviewer: "network is still there" won't be the case, since we're setting up a new adapter for scratch.
In most cases (assuming the user never set up a preferred Thread network before), there should be a new `ha-thread` prefixed Thread network (that the user can also choose to "Make primary Thread network"). 

Something like this:
```suggestion
   - **Result**: You should now see a new `ha-thread` Thread network.
```
